modules/nameable.rb

```diff
@@ -0,0 +1,5 @@
+class Nameable
+  def correct_name
+    raise NotImplementedError
```

Comment:
- [ ] Let's make this exception more descriptive for the user so that can know while class has the missing method.

  ```
  raise NotImplementedError, "#{self.class} has not implemented method '#{__method__}'"
  ```